### DISCUSSION TITLE
Overserr data source?

### DISCUSSION THREAD

**Discussion by trivarialthea** [sct/overseerr#3962] [Help]
Hi, 

Where does Overserr source its data from? For Bleach, for example, it shows 2 seasons:
![image](https://github.com/user-attachments/assets/fa94fbce-4a9e-4446-9d53-4fbd5977a039)

And that is completely different from what Sonarr sees and uses. https://www.thetvdb.com/series/bleach#seasons
![image](https://github.com/user-attachments/assets/63c26aa2-b3ce-4b9c-94c8-8685329404b6)
![image](https://github.com/user-attachments/assets/98f343d0-16a9-4fde-ab3b-5749dc87db85)

Is that a bug?

**Comment by dhaos21** [comment]
Probably https://www.themoviedb.org/tv/30984-bleach/seasons?language=en

**Comment by HimashaHerath** [comment]
is there a fix for this ?

**Reply by trivarialthea** [reply]
The only way to fix it seems to manually set the data on the backend (ie sonarr). Due to the policies that tmdb uses they won't change this (although it is factually wrong and I have noticed this on stuff like pantheon too)

**Reply by trivarialthea** [reply]
it would be nice to have an option to change the data source of overserr to thetvdb instead (even if it means providing your own api key).

**Reply by Lockszmith-GH** [reply]
While we're at it. It would be nice to be able to change it to a self-hosted "manual modifications" file. 
I find it strange that we all want to self-host our stuff because we don't trust the outside world, but design tools that draw exclusively from 3rd parties that we have no control over.
